examples/tts/configs/tacotron2.yaml

```diff
@@ -31,7 +31,7 @@ AudioToMelSpectrogramPreprocessor:
     n_fft: *n_fft
     frame_splicing: 1
     dither: 0.
-    feat_type: "logfbank"
+    #feat_type: "logfbank"
```

Comment:
please revert this change.  it helps clarity to keep it explicitly even if feat_type is "logfbank" by default.